.travis.yml

```
@@ -3,11 +3,8 @@ language: php
 sudo: false
 
 php:
-  - 7.2
   - 7.1
-  - 7.0
-  - 5.6
-  - 5.6
+  - 7.2
   - nightly
```

I think that `nightly` can be dropped as it is 8.0-dev now and instead 7.3 and 7.4 should be added:

```suggestion
  - 7.3
  - 7.4
```